src/platform_linux/platform.c

```diff
@@ -395,12 +398,14 @@ platform_histo_create(platform_heap_id       heap_id,
                       const int64 *const     bucket_limits,
                       platform_histo_handle *histo)
 {
+   platform_memfrag      memfrag_hh;
    platform_histo_handle hh;
-   hh = TYPED_MANUAL_MALLOC(
+   hh = TYPED_ARRAY_MALLOC(
```

Comment:
Equivalent calls.

Reply:
Actually, I believe the correct macro for this situation is `TYPED_FLEXIBLE_STRUCT_ZALLOC`.